packages/react/src/state.tsx

```diff
@@ -33,10 +33,7 @@ export function isStateUpdateAction<T extends JSONValue>(
   return action.name === 'state-update'
 }
 
-export function localStateExperimental<T extends JSONValue>(
-  initialValue: T,
-  key: string
-): StateModelState<T> {
+export function localStateExperimental<T>(initialValue: T, key: string): StateModelState<T> {
```

Comment:
I previously did this example using local state and these fixes were required. We're not using local state now, but will leave it there for the future! Basically, value can be anything, not just JSONValue. Otherwise, it was complaining when we included React Elements in the state.